css/styles.css

```diff
@@ -147,6 +191,32 @@ header {
   .bottom-contact-bar .bottom-icons {
     flex-direction: row;
   }
+
+  .mobile-modal-title {
+    display: none;
+  }
+
+  .desktop-modal-title {
+    font-size: 2rem;
+  }
+
+  .modal-tags li div {
+    font-size: 15px;
+    font-weight: 500;
+  }
+
+  @media screen and (max-width: 786px) {
+    button.modal-button {
+      font-size: 0.8rem;
+    }
+  }
+
+  .modal-details {
+    display: flex;
+    flex-direction: row;
```

Comment:
Please ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins).
Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/156456793-d5afe8b9-474c-4005-afb1-c7e604116c80.png) | ![image](https://user-images.githubusercontent.com/10332499/156456882-88e01d5e-dd9c-4af9-a10b-1289d5e87039.png)